src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/entities/MakeAnApplicationTypes.java

```diff
@@ -17,7 +17,8 @@ public enum MakeAnApplicationTypes {
     UPDATE_HEARING_REQUIREMENTS("Update hearing requirements"),
     UPDATE_APPEAL_DETAILS("Update appeal details"),
     REINSTATE("Reinstate an ended appeal"),
-    OTHER("Other");
+    OTHER("Other"),
```

Comment:
I would prefer to move this one above the OTHER type, as you know OTHER would come always at the end when they don't see anything that suits what they want.

Reply:
updated, thanks